### PR TITLE
Style for status bar + mode cycling, closes #88

### LINKED ISSUE #88: Layover for modes
It would be great to add a way to signal the current mode of the repl (now one only: example mode) to the user.

Maybe a stripe at the bottom like emacs.

### PR DESCRIPTION
**Pull request by tomasz-biernacki**
The gif explains it all:
kwlkpmjwgg
@arichiardi Now I wouldn't even bother @kajas90 for the images for parinfer... This nice keybord is perfect IMHO, and you have the mode below in the status bar, aka modeline.

### CODE REVIEW

File: src/cljs/cljs_repl_web/views.cljs [hunk 1]
```diff
@@ -148,7 +149,12 @@
   (let [media-query (subscribe [:media-query-size])
         console-created? (subscribe [:console-created? :cljs-console])
         example-mode? (subscribe [:queued-forms-empty? :cljs-console])
-        mode (subscribe [:get-console-mode :cljs-console])]
+        mode (subscribe [:get-console-mode :cljs-console])
+        modes (atom (cycle '(:paren-mode :indent-mode :none)))
```
Comment (tomasz-biernacki) [inline src/cljs/cljs_repl_web/views.cljs]: @arichiardi Not sure about the elegance of this one, but it seemed like a good idea.

Comment (arichiardi) [inline src/cljs/cljs_repl_web/views.cljs]: can't you save it in the state directly?

Comment (arichiardi) [inline src/cljs/cljs_repl_web/views.cljs]: Mmm no let me see better what it does...

Comment (tomasz-biernacki) [inline src/cljs/cljs_repl_web/views.cljs]: Yeah but it seemed like a technical implementation-detail. But yes, we can have it in the global state a create a handler for retrieving the next mode.

Comment (arichiardi) [inline src/cljs/cljs_repl_web/views.cljs]: If it is local state to the component I guess it is idiomatic even so yes

Comment (arichiardi) [inline src/cljs/cljs_repl_web/views.cljs]: The important is that the current state is in app-db globally, local can stay local for the logic, maybe a shared def for the vector of the three states will be needed in the future but not now I guess

Comment (tomasz-biernacki) [inline src/cljs/cljs_repl_web/views.cljs]: This `cycle`ed list is just to easily toggle between them, the current mode is already in the console global state under `:consoles (name console-key) :mode`

**Pull request merged**